docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,282 @@
+`tabbableElements` member variable.
+
+### `{inputMapper.base}.forwardTab(value)`
+
+- `value {Number}` Current value of the gamepad input.
+- Returns: Nothing.
+
+Changes the focus from the currently focused element to the next tabbable element using gamepad buttons and triggers.
+The next tabbable element is the element next to the currently focused element in the array stored as `tabbableElements`
+member variable.
+
+### `{inputMapper.base}.thumbstickTabbing(value, speedFactor, invert)`
+
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of tab navigation should be increased.
+- `invert {Boolean}` Whether the thumbstick tab navigation should be in opposite order (see below).
+- Returns: Nothing.
+
+Changes the focus from the currently focused element to the previous or next tabbable element using gamepad thumbsticks.
+Works only in **continuous** mode. Calls the [`{inputMapper.base}.reverseTab`](#inputmapperbasereversetabvalue) and
+[`{inputMapper.base}.forwardTab`](#inputmapperbaseforwardtabvalue) invokers according to the direction the thumbstick is
+pressed. For example, left on the horizontal axis and upward on the vertical axis of a thumbstick shifts the focus to
+previous DOM element. Pressing the thumbstick in opposite direction shifts the focus to the next DOM element.
+
+### `{inputMapper.base}.scrollLeft(value, speedFactor)`
+
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of left scrolling should be increased.
+- Returns: Nothing.
+
+Scrolls the web page left using gamepad buttons and triggers.
+
+### `{inputMapper.base}.scrollRight(value, speedFactor)`
+
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of right scrolling should be increased.
+- Returns: Nothing.
+
+Scrolls the web page right using gamepad buttons and triggers.
+
+### `{inputMapper.base}.scrollUp(value, speedFactor)`
+
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of upward scrolling should be increased.
+- Returns: Nothing.
+
+Scrolls the web page upward using gamepad buttons and triggers.
+
+### `{inputMapper.base}.scrollDown(value, speedFactor)`
+
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of downward scrolling should be increased.
+- Returns: Nothing.
+
+Scrolls the web page downward using gamepad buttons and triggers.
+
+### `{inputMapper.base}.scrollHorizontally(value, speedFactor, invert)`
+
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of horizontal scrolling should be increased.
+- `invert {Boolean}` Whether the horizontal scrolling should be in opposite order (see below).
+- Returns: Nothing.
+
+Scrolls the web page in left and right direction using gamepad thumbsticks. Calls the
```

Comment:
I missed this variation before, this should just be "scrolls the page left and right using the gamepad thumbsticks".